NRediSearch/Query.cs

```diff
@@ -260,5 +277,18 @@ public Query LimitFields(params string[] fields)
             this._fields = fields;
             return this;
         }
+
+        /// <summary>
+        /// Set the query to be sorted by a sortable field defined in the schema
+        /// </summary>
+        /// <param name="field">the sorting field's name</param>
+        /// <param name="ascending">if set to true, the sorting order is ascending, else descending</param>
+        /// <returns>the query object itself</returns>
+        public Query SetSortBy(string field, bool ascending)
```

Comment:
recommend: `public Query SetSortBy(string field, bool ascending = true)` - i.e. make it optional with a default of ascending